app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+
+<main>
+  <% unless @media_spotlight.nil? %>
+    <h2>Media Spotlight: <%= @media_spotlight.title%> by <%= @media_spotlight.creator%> </h2>
+  <% else %>
+    <h2>No Media Spotlight</h2>
+  <% end %>
+  
+  <%# MOVIES %>
+  <h2>Top Movies</h2>
+  <section>
+   <% if @top_movies.count > 0 %>
+      <ul>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?